src/pages/Homepage/HeroSlideshow.vue

```diff
@@ -1,182 +1,128 @@
-					A Monthly Good subscription<br class="su">
-					makes supporting entrepreneurs<br class="su">
-					part of your routine with monthly contributions.
-				</p>
-			</div> -->
-
-			<div class="headline billion-to-women-headline" v-if="promoEnabled">
-				<span class="headline-title">We just hit $1 billion in loans to women.</span>
-				<p class="headline-body">
-					To celebrate, we'll give you a free<br class="su">
-					$25 credit after you make a loan this week.
-				</p>
-			</div>
-			<div class="headline" v-else>
-				<span class="headline-title">Dreams are universal,<br class="smo"> opportunity is not.</span>
-				<p class="headline-body">
-					Lend as little as $25<br class="so">
-					to create<br class="mo">
-					opportunity<br class="so"><br class="lu">
-					for people<br class="mo">
-					around the world.
-				</p>
-			</div>
-			<div class="mask">
-				&nbsp;
-			</div>
-		</div>
-		<div class="action-button-wrapper">
-			<!-- -------------
-			Uncomment when the Billion To Women Campaign ends on Oct. 13th 2019
-			-------------------- -->
-
-			<!-- <kv-button
-				v-if="showMgPromo"
-				class="slideshow-action-button"
-				to="/monthlygood"
-				v-kv-track-event="['Home', 'EXP-Montly-Good-Promo', 'click-Sign-up']"
-			>
-				Sign up
-				<span v-if="showDoubleArrowButton"> >></span>
-			</kv-button> -->
-
-			<kv-button
-				v-if="promoEnabled"
-				class="slideshow-action-button billion-to-women-action"
-				to="/lend/lendingReward?lending_reward=265&doneUrl=lend-by-category/women"
-				v-kv-track-event="['Home', 'Lend now', 'click-Promo 1 Billion to Women 2019']"
-			>
-				Lend now
-				<span v-if="showDoubleArrowButton"> »</span>
-			</kv-button>
-			<kv-button
-				v-else
-				class="slideshow-action-button"
-				to="/lend-by-category"
-				v-kv-track-event="['Home', 'EXP-HeroWhyKiva', 'click-Start-lending', null, 'true']"
-			>
-				Start lending
-				<span v-if="showDoubleArrowButton"> »</span>
-			</kv-button>
-		</div>
+	<div>
+		<!-- Promo -->
+		<kv-hero v-if="promoEnabled" headline-bg-color="#00244e" class="possibilities-promo">
+			<template v-slot:images>
+				<kv-carousel @change="slideChange">
+					<kv-carousel-slide>
+						<kv-responsive-image :images="possibilitiesImages(1)" />
+					</kv-carousel-slide>
+					<kv-carousel-slide>
+						<kv-responsive-image :images="possibilitiesImages(2)" />
+					</kv-carousel-slide>
+					<kv-carousel-slide>
+						<kv-responsive-image v-if="counter > 2" :images="possibilitiesImages(3)" />
+					</kv-carousel-slide>
+				</kv-carousel>
+			</template>
+			<template v-slot:headlineTitle>
+				Make opportunity possible <br class="smo">for borrowers around the world.
+			</template>
+			<template v-slot:action>
+				<div class="row">
+					<div class="small-12 large-6 columns action-1">
+						<kv-button
+							class="action-button"
+							to="/lend-by-category/"
+							v-kv-track-event="['Home', 'possibility', 'click-Lend-now', null, 'true']"
```

Comment:
These are the legacy params. You can see in src/plugins/kv-analytics-plugin.js that v-kv-track-event only takes 4 params.  this should be v-kv-track-event="['Home', 'possibility', 'click-Lend-now'] with the value param null. 'true' shouldn't be a param. Also applies to other v-kv-track events on this page